lib/resources/r4/allergy_intolerance.yaml

```diff
@@ -311,7 +296,6 @@ fields:
 
 - name: onset_x_
```

Comment:
can you check quickly if there's a reason we aren't using square brackets here? might need to escape the characters

Reply:
based on other files plain old unescaped square brackets seem to work, at least in a terminology table

Reply:
[I squared up.](https://github.com/cerner/fhir.cerner.com/pull/374/commits/6fb0b4a5a1bdcd0b07e086f542d44ded83b12430?file-filters%5B%5D=.yaml#diff-eeda3e608dc0db5d6d7c877900383353L297) I also added square brackets [here](https://github.com/cerner/fhir.cerner.com/pull/374/commits/6fb0b4a5a1bdcd0b07e086f542d44ded83b12430?file-filters%5B%5D=.yaml#diff-abb99d2139d79f8a2a8f1d214f5f6882L301) and [here.](https://github.com/cerner/fhir.cerner.com/pull/374/commits/6fb0b4a5a1bdcd0b07e086f542d44ded83b12430?file-filters%5B%5D=.yaml#diff-abb99d2139d79f8a2a8f1d214f5f6882L316)